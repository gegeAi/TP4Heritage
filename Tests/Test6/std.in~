S segment 0 1 2 3
R rectangle 5 6 8 9
PC poly 2 3 5 6 7 8 9 6
OR reunion segment rectangle
OI inter segment rectangle
HIT segment 0 1
HIT rectangle 5 6
HIT PC 2 3
HIT reunion 0 1
HIT inter 0 1
