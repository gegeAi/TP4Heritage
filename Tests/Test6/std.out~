OK
#New object: segment
OK
#New object: rectangle
OK
#New object: poly
