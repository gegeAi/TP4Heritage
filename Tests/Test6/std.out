OK
#New object: segment
OK
#New object: rectangle
OK
#New object: poly
YES
YES
YES
YES
YES
